src/script/common/c_internal.h

```diff
@@ -57,6 +57,9 @@ enum {
 	CUSTOM_RIDX_BACKTRACE,
 	CUSTOM_RIDX_HTTP_API_LUA,
 	CUSTOM_RIDX_METATABLE_MAP,
+
+	// The following four functions are implemented in Lua because LuaJIT can
+	// trace them and make them faster than using the Lua C API to do the same.
```

Comment:
```cpp
	// The following four functions are implemented in Lua because LuaJIT can
	// trace them and optimize tables/string better than from the C API.
```